action.yml

```
@@ -32,5 +32,5 @@ inputs:
     required: false
     description: "The package manager used to run the build and install commands. If not provided, the manager will be auto detected. Example values: `yarn`, `npm`, `pnpm`."
 runs:
-  using: 'node12'
+  using: 'node16'
```

```suggestion
  using: 'node20'
```
https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/

---

Good idea ! I didn't want to go too fast 😉